ezpublish/config/config.yml

```diff
@@ -19,7 +19,7 @@ framework:
     # Place "eztpl" engine first intentionnally.
     # This is to avoid template name parsing with Twig engine, refusing specific characters
     # which are valid with legacy tpl files.
-    templating:      { engines: ['eztpl', 'twig'] } #assets_version: SomeVersionScheme
+    templating:      { engines: ['twig'] } #assets_version: SomeVersionScheme
```

Comment:
hmm, then maybe better here so people easily see it, and so we don't change location.